src/riak_test_escript.erl

```diff
@@ -149,6 +149,7 @@ main(Args) ->
                     Tests0;
                 {_, undefined} ->
                     Tests0;
+                %% TODO don't understand what this does
```

Comment:
Did you want to add this comment?  I can explain (more) of what this is meant to do.  When `riak_test` was written it was supposed to run a whole bunch of tests sequentially and this shuffles the order.  Not that useful now that we have independent tests on giddyup.  It was meant to break any potential test interdependence.